Publications (#2735)
nf-core/website: sites/docs/src/content/docs/guidelines/publication_credit.md

---
title: Publication credit
subtitle: Recognition for nf-core pipeline publications
weight: 123
---

> All pipeline publications and presentations, or those that describe pipelines for the first time, should give attribution to the nf-core community.

Reviewer: ```suggestion
All pipeline publications and presentations, or those that describe pipelines, should give attribution to the nf-core community.
```

> We would also like to thank the nf-core team, in particular Harshil Patel, Phil Ewels and Maxime Garcia, for welcoming us to the nf-core community and their assistance in reviewing and releasing the pipeline. We would also like to thank the nf-core community for developing the nf-core infrastructure and resources for Nextflow pipelines. A full list of nf-core community members is available at https://nf-co.re/community.

**Optionally**, the nf-core community can be included as a consortium co-author. This option is preferred when a pipeline has made extensive use of existing nf-core pipeline components (e.g., modules and subworkflows) written by other community members who were not directly involved in the pipeline itself. For example, see hgtseq: [A Standard Pipeline to Study Horizontal Gene Transfer](https://doi.org/10.3390/ijms232314512).

If members of the nf-core community have provided significant input to the creation or maintenance of a pipeline, please consider adding them as coauthors on the pipeline publication or presentation.

> If your pipeline publication or presentation is an extension of a community pipeline, please consider adding the main developers as coauthors.

Reviewer: ```suggestion
If your pipeline publication or presentation is an extension of a community pipeline, please discuss authorship with the main developers.
```
